app/services/audit/helper.rb

```diff
@@ -0,0 +1,7 @@
+module Audit
+  module Helper
+    def instrument_name(name)
+      "#{name}#{Audit::Config::NOTIFICATION_SUFFIX}"
```

Comment:
`NOTIFICATION_SUFFIX` is used only here and in the tests, shouldn't be a constant of this module?